sources/us/ny/statewide.json

```diff
@@ -2399,11 +2399,11 @@
         "addresses": [
             {
                 "name": "state",
-                "website": "https://gis.ny.gov/gisdata/inventories/details.cfm?DSID=921",
-                "data": "https://data.openaddresses.io/cache/uploads/iandees/05d0aa/OpenAddresses_NYS_20230110.gdb.zip",
+                "website": "https://data.gis.ny.gov/datasets/sharegisny::nys-address-points/about",
+                "data": "https://data.openaddresses.io/cache/uploads/iandees/f2b763/OpenAddresses_NYS_20230331.gdb.zip",
```

Comment:
This URL returns a 404

Reply:
Thanks, I had copied it to the wrong location. Fixed that and will try re-running.